### PR TITLE
Fix typos in README

### PR DESCRIPTION
**Pull request by lARSHADl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,9 +99,9 @@ Note: Instead of creating a .env file, you can just run this command "cp env.exa
   REACT_APP_GITHUB_TOKEN = "YOUR GITHUB TOKEN HERE"
 ```
 
-Set `showGithubProfile` to true or false to show Contact Profile using Github, defaults to false.
+Set `showGithubProfile` to true or false to show Contact Profile using Github,Default is set as false.
```
Comment (naveen521kk) [inline README.md:102]: I think the previous one correct.

**Review by naveen521kk — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -99,9 +99,9 @@ Note: Instead of creating a .env file, you can just run this command "cp env.exa
...
 
-Note: Open Source Projects section only show pinned itens of your Github. If you are seeing [this](https://i.imgur.com/Hj6mu1K.png), please follow this
+Note: Open Source Projects section only show pinned items of your Github. If you are seeing [this](https://i.imgur.com/Hj6mu1K.png), please follow this
```
Comment (naveen521kk) [inline README.md:104]: **Suggestion**: Embed the image instead of saying, if you see this.

**Review by naveen521kk — CHANGES_REQUESTED** [review]
Sorry for delay :(

File: README.md (hunk 2 above)
Comment (kartikcho) [inline README.md:104]: Change `follow this` to `follow these`.

File: README.md [hunk 3]
```diff
@@ -99,9 +99,9 @@ Note: Instead of creating a .env file, you can just run this command "cp env.exa
...
 [Instructions](https://docs.github.com/en/enterprise/2.13/user/articles/pinning-items-to-your-profile).
```
Comment (kartikcho) [inline README.md:105]: Remove capitalized `I` in Instructions.

**Review by kartikcho — CHANGES_REQUESTED** [review]

**Comment by naveen521kk** [comment]
Any update on this @lARSHADl ?

**Comment by lARSHADl** [comment]
Sorry guys new to github and this was my first commit

File: README.md (hunk 1 above)
Comment (naveen521kk) [reply README.md:102]: @lARSHADl Could you revert this to how it was before?

**Comment by lARSHADl** [comment]
Done !

-------- Original Message --------

File: README.md [hunk 4]
```diff
@@ -101,7 +101,12 @@ Note: Instead of creating a .env file, you can just run this command "cp env.exa
 
 Set `showGithubProfile` to true or false to show Contact Profile using Github, defaults to false.
 
-Note: Open Source Projects section only show pinned itens of your Github. If you are seeing [this](https://i.imgur.com/Hj6mu1K.png), please follow this
+Note: Open Source Projects section only show pinned items of your Github. If you are seeing it as show below 
+
+
+<img src = "https://i.imgur.com/Hj6mu1K.png" alt = " Error" > 
+
```
Comment (naveen521kk) [inline README.md:108]: Why not use markdown like
```suggestion
![Error](https://i.imgur.com/Hj6mu1K.png)
```

File: README.md [hunk 5]
```diff
@@ -101,7 +101,12 @@ Note: Instead of creating a .env file, you can just run this command "cp env.exa
...
+Please follow these 
 [Instructions](https://docs.github.com/en/enterprise/2.13/user/articles/pinning-items-to-your-profile).
```
Comment (naveen521kk) [inline README.md:110]: As @kartikcho said do 
```suggestion
[instructions](https://docs.github.com/en/enterprise/2.13/user/articles/pinning-items-to-your-profile).
```

**Comment by lARSHADl** [comment]
Will change to the last check point with ! Markdown .

**Review by kartikcho — APPROVED** [review]
Hey @lARSHADl, thanks for opening a PR for this, everything looks good now!

**Comment by lARSHADl** [comment]
Is there anything that's bothering you, please feel free to suggest  ?

**Review by naveen521kk — APPROVED** [review]
Sorry for the delay!

**Comment by naveen521kk** [comment]
@all-contributors please add @lARSHADl for doc

**Pull request merged**